Content.Server/Medical/HealthAnalyzerSystem.cs

```diff
@@ -21,17 +26,21 @@ public sealed class HealthAnalyzerSystem : EntitySystem
         [Dependency] private readonly SharedDoAfterSystem _doAfterSystem = default!;
         [Dependency] private readonly SolutionContainerSystem _solutionContainerSystem = default!;
         [Dependency] private readonly UserInterfaceSystem _uiSystem = default!;
+        [Dependency] private readonly TransformSystem _transformSystem = default!;
 
         public override void Initialize()
         {
             base.Initialize();
             SubscribeLocalEvent<HealthAnalyzerComponent, AfterInteractEvent>(OnAfterInteract);
             SubscribeLocalEvent<HealthAnalyzerComponent, HealthAnalyzerDoAfterEvent>(OnDoAfter);
+            SubscribeLocalEvent<HealthAnalyzerComponent, DroppedEvent>(OnDropped);
+            SubscribeLocalEvent<HealthAnalyzerComponent, EntGotInsertedIntoContainerMessage>(OnInsertedIntoContainer);
```

Comment:
the handlers of these 2 need to use `Enitity<>`, while the events arent raised by ref you can just put ref in the param as normal to avoid the error it throws. Also, try to order the handlers in the file in the same order you subscribe them (OnPowerCellSlotEmpty is above OnDropped, for example)